AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java

```diff
@@ -105,6 +106,25 @@
 
     // Not in libanki
     protected WeakReference<Activity> mContextReference;
+    /**
+     * The card currently being reviewed.
+     *
+     * Should be skipped is found to be the next card. This allow to
```

Comment:
Consider condensing both lines to:
```suggestion
     * Must not be returned during prefetching (as it is currently shown).
```

Reply:
Done